application/views/dashboard/index.php

```diff
@@ -167,10 +178,10 @@ function echo_table_col($row, $name) {
 
     		<thead>
 				<tr class="titles">
-					<th><?= __("Date"); ?></th>
-
+					<!-- Display UTC or Local in header based on settings -->
+					<th><?= __("Date"); ?><?php if($this->session->userdata('user_show_log_utc') == 1){ echo __(" (UTC)"); }else{ echo __(" (Local)"); } ?></th>
```

Comment:
Now there are coming a lot of views which display the new time.

This could be omitted. Yeah it's nice to display that but I'd rather generate this in the Controller to avoid redudant code